### PR TITLE
refactor(getter): Never throw an UnsupportedOpException in getter.

### LINKED ISSUE #436: Removes UnsupportedOperationException in getter of the AST

### PR DESCRIPTION
**Pull request by GerardPaligot**
Closes #436

### CODE REVIEW

File: src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ public boolean removeThrownType(CtTypeReference throwType) {
 
 	@Override
 	public String getSimpleName() {
-		throw new UnsupportedOperationException("You can't have a name in a anonymous executable");
+		return "";
```
Comment (monperrus) [inline src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java]: could you add test to specify this?

File: src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java [hunk 2]
```diff
@@ -155,7 +155,7 @@ public String getSimpleName() {
 
 	@Override
 	public CtTypeReference<Void> getType() {
-		throw new UnsupportedOperationException("An anonymous executable isn't typed");
+		return factory.Type().VOID_PRIMITIVE;
```
Comment (monperrus) [inline src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java]: idem?

Comment (GerardPaligot) [inline src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java]: Done

File: src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java (hunk 1 above)
Comment (GerardPaligot) [inline src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java]: Done

**Pull request merged**